### PR TITLE
Add todos ;)

### PR DESCRIPTION
**Pull request by evgeniidatsiuk**

### CODE REVIEW

**Comment by evgeniidatsiuk** [comment]
Hello @yevhenprudnik, could you fix these todos?)

File: services/post-service.js [hunk 1]
```diff
@@ -12,8 +13,15 @@ class PostService {
...
-
+  // TODO: example of documentation
+  /**
+   * Applies paginated filters to the posts.
+   *
+   * @param userId
+   *   User id.
+   * @param postedBy
+   *   Author id.
+   */
```
Comment (evgeniidatsiuk) [inline services/post-service.js:24]: _hint:_ there also must be pagination object that include skip, limit and page or **you can use cursor pagination**

File: route/post-route.js [hunk 2]
```diff
@@ -3,8 +3,18 @@ const postRouter = new Router();
 const postController = require('../controllers/post-controller');
 const authMiddleware = require('../middleware/auth-middleware');
 
+// TODO: use as global authMiddleware, don't use for every route
+
 postRouter.get('/', (req, res) => {res.json("router's working")});
 
+// TODO: .post(/posts, postController.createPost)
+
+// TODO: .post(/posts/:id/comments, postController.commentPost)
```
Comment (evgeniidatsiuk) [inline route/post-route.js:12]: Also I recommend move it to comment controller and service ;)

**Comment by yevhenprudnik** [comment]
Hello @evgeniidatsiuk, yes, sure, I'll fix that

**Pull request merged**